test/card/card.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('Directive: pfCard', function() {
```

Comment:
Shouldn't the name of this file match the name of the directive? I would think this should be named `pf-card.spec.js`.